frontmacs-javascript.el

```diff
@@ -17,4 +18,10 @@
 (custom-set-variables '(js-indent-level 2)
                       '(js2-basic-offset 2))
 
+;; setup jsdoc
+(add-hook 'js2-mode-hook
+           #'(lambda ()
+               (define-key js2-mode-map "\C-ci" 'js-doc-insert-function-doc)
+               (define-key js2-mode-map "@" 'js-doc-insert-tag)))
```

Comment:
The first sharp quote is redundant, but the other functions could use them:

```elisp
(add-hook 'js2-mode-hook
           (lambda ()
               (define-key js2-mode-map "\C-ci" #'js-doc-insert-function-doc)
               (define-key js2-mode-map "@" #'js-doc-insert-tag)))
```

http://endlessparentheses.com/get-in-the-habit-of-using-sharp-quote.html is a great read on when to use sharp quote and when not.